### PR TITLE
jumpserver alert definition

### PR DESCRIPTION
**Pull request by viennaa**

### CODE REVIEW

File: system/kube-monitoring/charts/prometheus-frontend/openstack-jump.alerts [hunk 1]
```diff
@@ -47,3 +47,15 @@ groups:
...
       description: 'Datapath {{ $labels.service }} {{ $labels.check }} is flapping for 30 minutes. See Sentry for details'
       summary: 'Datapath {{ $labels.service }} {{ $labels.check }} is flapping'
+
+  - alert: JumpserverRSAForwardingDown
+    expr: jump_rsa_forwarding_up == 1 or absent(jump_rsa_forwarding_up)
+    for: 15m
+    labels:
+      context: availability
+      service: jumpserver
+      severity: warning
```
Comment (auhlig) [inline system/kube-monitoring/charts/prometheus-frontend/openstack-jump.alerts]: Suggesting to start with `info` level to test and tune the alert for some days. Feel free to adjust the severity to `warning` after testing.

Comment (viennaa) [reply system/kube-monitoring/charts/prometheus-frontend/openstack-jump.alerts]: Good idea.

**Comment by auhlig** [comment]
The alert would show in the `#alert-os-<severity>` and `#alert-jumpserver-<severity>` (if that one already exists; create channel?) slack channel. Feel free to change the `service` label if you want to have it in another slack channel.

**Comment by viennaa** [comment]
Can I omit tier and service as it just seems slack relevant?

**Comment by auhlig** [comment]
It's relevant for routing alerts to the respective receivers (by the `tier`, `service` and `severity` label).
If you omit them your alert will be routed to nirvana.

**Pull request merged**